tests/acceptance/stepDefinitions/sharingContext.js

```diff
@@ -880,6 +880,16 @@ Then('{string} {string} should be listed in the autocomplete list on the webUI',
   )
 })
 
+Then(
+  'the share permission denied message should be displayed in the sharing dialog on the webUI',
+  async function() {
+    const permissionDeniedMessage = "You don't have permission to share this folder."
```

Comment:
why not getting this from feature?

Reply:
that part of the ui doens't shows any other type of message. Its only for the permission denied (as far as I have seen) so it makes more sense to make specific step for that.